README.md

```diff
@@ -1,36 +1,134 @@
+adding a certificate to your truststore, make sure that you **trust it and all certificates that are signed by it to exchange payments
+with your financial institution**. It is recommended that each financial institution uses a self-signed certificate, and that each financial
+institution stores the self-signed certificate of each financial institution that they trust in their truststore. Trusting root certificate CAs
+commonly used on the web **should be avoided**.
+
+Generating a key- and truststore can be accomplished using for example [keytool](https://docs.oracle.com/cd/E19798-01/821-1841/gjrgy/).
+If you use that page as a guide however it is **not** recommended to use the `-trustcacerts` flag. Furthermore, it is recommended to
+use the PKCS12 file format (`.p12` file extension) for key- and truststores instead of the older JKS format. (`.jks` file extension)
 
-## Configuration
+### 6.3 Sending Messages
+To send a message, start by creating a `pacs.008` message including a business header, such as `head.001`.
+This message header should contain the hostnames of the sender and the recipient of the message in its `Fr` and
+`To` tags. An example is available below.
+```xml
+<h:AppHdr xmlns:h="urn:iso:std:iso:20022:tech:xsd:head.001.001.02">
+    <h:Fr>
+        <h:FIId>
+            <h:FinInstnId>
+                <!--The sending financial institution has the hostname "example.org" and its external API runs on port 443-->
+                <h:Nm>example.org</h:Nm>
+            </h:FinInstnId>
+        </h:FIId>
+    </h:Fr>
+    <h:To>
+        <h:FIId>
+            <h:FinInstnId>
+                <!--The receiving financial institution has the hostname "example.com" and its external API runs on port 1234-->
+                <h:Nm>example.com:1234</h:Nm>
+            </h:FinInstnId>
+        </h:FIId>
+    </h:To>
+    <!--Further tags here...-->
+</h:AppHdr>
+<!--ISO 20022 message here...-->
+```
+The `Fr` tag must contain the hostname of your financial institution. Specifically, it must be a subject alternative name of your
+personal X.509 certificate. The `To` tag will be used by the microservice to determine the financial institution the message should
+be sent to. The port numbers of both the sending and receiving financial institution's external APIs can optionally be specified, but
+if they are not present they are assumed to be located on port 443.
+
+After the message is constructed, it should be sent to your financial institutions internal API by sending a HTTP request using the
+POST method to its `/api/v1/pacs` endpoint. The response can either be a `pacs.002` message indicating success, or an error message.
+An example of an error message can be found below. For a list of all error statuses that can be returned by the internal and
+external APIs, see the [OpenAPI description](OpenAPI/decentralized-iso20022-v1.yaml)
```

Comment:
Is this something the financial institution will have to implement? They can either choose to implement `pacs.002` or HTTP error messages? Or is there some sort of configuration for this?